src/renderer/src/hooks/useUtilitySelector.ts

```diff
@@ -108,7 +89,7 @@ export const useMigrationsNeo3Selector = () => {
 
 export const useMigrationNeo3Selector = (hash: string) => {
   const { value: migrationNeo3, ref: migrationNeo3Ref } = useAppSelector(
-    ({ utility }) => utility.data.migrationsNeo3[UtilsHelper.normalizeHash(hash)]
+    ({ utility }) => utility.data.migrationsNeo3[BSTokenHelper.normalizeHash(hash)]
```

Comment:
I think it can break the old migrations. Because now the `0x` is add, and it was saved without `0x` before.